pkg/generators/forwarding/fluentd/output_conf_forward_test.go

```diff
@@ -0,0 +1,125 @@
+package fluentd
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	test "github.com/openshift/cluster-logging-operator/test"
+)
+
+var _ = Describe("Generating fluentd secure forward output store config blocks", func() {
+
+	var (
+		outputs   []logging.OutputSpec
+		generator *ConfigGenerator
+	)
+	BeforeEach(func() {
+		var err error
+		generator, err = NewConfigGenerator()
+		Expect(err).To(BeNil())
+	})
+
+	Context("for a secure endpoint", func() {
+		BeforeEach(func() {
+			outputs = []logging.OutputSpec{
+				{
+					Type:     logging.OutputTypeForward,
+					Name:     "secureforward-receiver",
+					Endpoint: "es.svc.messaging.cluster.local:9654",
+					Secret: &logging.OutputSecretSpec{
+						Name: "my-infra-secret",
+					},
+				},
+			}
+		})
+
+		It("should produce well formed output label config", func() {
+			results, err := generator.generateOutputLabelBlocks(outputs)
+			Expect(err).To(BeNil())
+			Expect(len(results)).To(Equal(1))
+			test.Expect(results[0]).ToEqual(`<label @SECUREFORWARD_RECEIVER>
+	<match **>
+		# https://docs.fluentd.org/v1.0/articles/in_forward
+	   @type forward
+	   <security>
+	     self_hostname "#{ENV['NODE_NAME']}" 
```

Comment:
This has a trailing whitespace; probably why the unit tests fails.

Reply:
we dont compare leading or trailing white space.  I added in a normalizer because I couldn't get the formatting right.